delfin/api/v1/storages.py

```diff
@@ -283,3 +255,19 @@ def _set_synced_if_ok(context, storage_id, resource_count):
         storage['sync_status'] = resource_count * constants.ResourceSync.START
         storage['updated_at'] = current_time
         db.storage_update(context, storage['id'], storage)
+
+
+def _create_performance_monitoring_task(context, storage_id, capabilities):
+    # check resource_metric attribute availability and
+    # check if resource_metric is empty
+    if 'resource_metrics' not in capabilities \
+            or not bool(capabilities.get('resource_metrics')):
+        raise exception.EmptyResourceMetrics()
+
+    task = dict()
+    task.update(storage_id=storage_id)
+    task.update(args=capabilities.get('resource_metrics'))
+    task.update(interval=constants.Task.DEFAULT_TASK_INTERVAL)
```

Comment:
hope this has to be inn sync with scheduler , better to take from conf file.

Reply:
Yes agreed, That's why added as constant. Will fix it in scheduler PR